articles/architecture-scenarios/b2c/authentication.md

```diff
@@ -0,0 +1,120 @@
+
+Nearly every B2C application provides that ability for their customers to create a new set of credentials for this company. This is a common form of authentication that all users are familiar with.
+
+Username Password Authentication can come in multiple flavors at Auth0. If your application is a greenfield application with no existing user base, then a simple Auth0 OOTB database connection will give you everything you need to start authenticating your users. However if you have a legacy user store (such as your own database of users or an existing LDAP system) you have a couple of different options for migrating your users, see here for more information.
+
+However you end up provisioning the users for your database connection, the authentication of those users is quite similar. It requires you to present the users with a form to enter their username and password. As mentioned in the universal login section, the simplest and safest way to authenticate users with a username and password is to redirect them to a centralized login page and collect their username and password there. This allows your authorization server to determine whether they have already authenticated and skip the login form entirely when it is not needed. See here for more information.
+
+::: note
+Collecting credentials only at the centralized login page will reduce the surface area for a potential leak of user secrets. It will also reduce the need to collect credentials unnecessarily.
+:::
+
+## Anomaly Detection
+
+The reason that authentication systems are important is to prevent bad actors from accessing applications and user data that they should not. We want to place as many barriers in between those bad actors and access to our system as possible. One of the easiest ways to do this is to ensure that your anomaly detection with Auth0 is configured correctly. Take a moment to [read the docs on anomaly detection](/anomaly-detection) and ensure it is working correctly for you behind the scenes.
+
+::: note
+Anomaly Detection is handled by Auth0 and provides a great security feature for your product. If you are going to utilize it, ensure that you have set up your SMTP provider and configured your email templates before turning on email delivery to your users.
+:::
+
+## Application Integration
+
+Once you have figured out how you want to authenticate your users, the next step is to determine how and when you will initiate that authentication. Each application may have its own starting point.
+
+::: warning
+Native Mobile Applications (and desktop applications) should use the system browser for authentication, or they open themselves up to additional security risks.
+:::
+
+As discussed, OpenID Connect (OIDC) is the most frequently used industry standard protocol when it comes to customer facing applications. Figuring out which OIDC flow to use is your first task, and you will want to start by reviewing guidance concerning the various grant types here.  
```

Comment:
My comment there was more related to our experience with Auth0 customers and not an industry wide data point.  Perhaps it can just be reworded:
```suggestion
As discussed, we have found that most Auth0 customers use OpenID Connect (OIDC) as their industry standard protocol when it comes to customer facing applications. Figuring out which OIDC flow to use is your first task, and you will want to start by reviewing guidance concerning the various grant types here.  
```

Reply:
Sounds good, though I was more referring to the "As discussed" bit at the start of the paragraph. It would probably be worthwhile adding a link to the "where" it was discussed :-)

Reply:
Added link to grant type mapping